components/support/locale/src/main/java/mozilla/components/support/locale/LocaleManager.kt

```diff
@@ -92,6 +93,7 @@ object LocaleManager {
         }
     }
 
+    @SuppressLint("AppBundleLocaleChanges")
     private fun updateConfiguration(context: Context, locale: Locale): Context {
```

Comment:
Should we create a follow up ticket ? For addressing the suppression :)

Reply:
I think this is a false positive because AFAIK we do not ship app bundles with split languages and we do not need to download additional languages when changing locales (this is what the warning is about).

Maybe someone more familiar with what we ship on Play Store can correct me if I am wrong.
If not, there is nothing to address here, just supressing the warning (maybe I can also add a comment about it)

Reply:
This is correct. I looked into this during my pass as well. :)